cmd/juju/controller/destroy.go

```diff
@@ -75,6 +78,18 @@ If there is persistent storage in any of the models managed by the
 controller, then you must choose to either destroy or release the
 storage, using ` + "`--destroy-storage` or `--release-storage` respectively." + `
 
+Sometimes, the destruction of a hosted model may fail as Juju encounters errors
+that need to be dealt with before that model can be destroyed.
+However, at times, there is a need to destroy a controller ignoring
+such model errors. In these rare cases, use --force option but note 
+that --force will also remove all units of any hosted applications, their subordinates
+and, potentially, machines without given them the opportunity to shutdown cleanly.
+
+Model destruction is a multi-step process. Under normal circumstances, Juju will not
+proceed to a next step until the current step has finished. 
```

Comment:
... proceed to *the* next step

Reply:
also tweaked the several other places where this text is used